packages/node_modules/@ciscospark/react-component-file-share-display/src/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ShareFile from '@ciscospark/react-component-activity-share-file';
+import ShareThumbnail from '@ciscospark/react-component-activity-share-thumbnail';
+
+const defaultProps = {
+  isPending: false,
+  objectUrl: ''
+};
+
+const propTypes = {
+  file: PropTypes.object.isRequired,
+  isFetching: PropTypes.bool.isRequired,
+  isPending: PropTypes.bool,
+  objectUrl: PropTypes.string,
+  onDownloadClick: PropTypes.func.isRequired
+};
+
+function FileShareDisplay(props) {
+  const {
+    file,
+    isFetching,
+    isPending,
+    objectUrl,
+    onDownloadClick
+  } = props;
```

Comment:
We should probably use destructuring here since the actual `props` object isn't used anywhere. i.e.:
``` js
function FileShareDisplay({
  file,
  isFetching,
  isPending,
  objectUrl,
  onDownloadClick
}) {
// blah blah blah
}
```